.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: node_js
 node_js:
-  - 6
   - 8
+  - 10
 cache:
   directories:
     - node_modules
 script:
   - npm test
-  - npm run build
+  - npm run build
```

Comment:
@jkingoliver @joyychang Y'all should run some `awk` upstream to make sure all generated text files have proper newlines at the end of every file -it's a constant pet peeve of mine with all the generated code!